pom.xml

```diff
@@ -607,10 +607,10 @@
         <commons-logging.version>1.1.1</commons-logging.version>
 
         <dnsjava.version>2.1.8</dnsjava.version>
-        <junit.version>4.12</junit.version>
-        <junit.jupiter.version>5.2.0</junit.jupiter.version>
-        <junit.plateform.version>1.4.1</junit.plateform.version>
-        <junit.vintage.version>5.2.0</junit.vintage.version>
+        <junit.version>4.13-beta-3</junit.version>
```

Comment:
Not sure "beta" is a good idea...

Reply:
so, the last final version is still 4.12. I will drop that commit